modules/collector-source/pkg/metric/diagnostics_test.go

```diff
@@ -44,7 +48,7 @@ func TestDiagnosticsModule_Update(t *testing.T) {
 		t.Error("Expected node diagnostic to pass")
 	}
 
-	kubecostDetails, err := module.DiagnosticsDetails(KubecostDiagnosticMetricID)
+	kubecostDetails, err := module.DiagnosticsDetails(OpencostDiagnosticMetricID)
```

Comment:
```suggestion
	opencostDetails, err := module.DiagnosticsDetails(OpencostDiagnosticMetricID)
```
if we want to remove these references too? though not that much important